### ISSUE TITLE
Add link to improve the map base layer

### ISSUE THREAD

**Issue by rjmackay** [ushahidi/platform#1632]
Add a link to improve the OSM base data https://www.mapbox.com/blog/improve-this-map/

**Comment by Shadrock** [comment]
To clarify this issue: When I initially had the conversation w/ Mapbox the goal was to provide users with a quick, easy, way to identify any issues in the underlying OSM database without necessarily having to go into OSM edit mode and make the changes themselves, which the blog post linked above seems to suggest. 

We wanted to pursue the "Improve this map" link that's used for _feedback_ and not direct editing ([Mapbox blog post here](https://www.mapbox.com/blog/map-feedback/)). This feature is already used around the internet, including on the Github previews of GeoJSON as seen below: 

![improve osm link](https://cloud.githubusercontent.com/assets/4574998/24114248/28df2798-0d65-11e7-8c38-b23fb2c9c119.png)

The URL is dynamically linked is the [Mapbox Map Feedback Page](https://www.mapbox.com/map-feedback), which takes the user to the same zoom level and lat/long they were looking at. Mapbox owns triaging and responding to issues logged via this feedback page and makes the appropriate edits in OSM. 

I am logging a separate issue to respond to previous discussions relating to expanding our help section to include information for editing OSM for use as a base layer in Ushahidi.

**Comment by jshorland** [comment]
We will likely need to add help text -- "Clicking this link will take you to... blah blah blah"

**Comment by Shadrock** [comment]
Yep. I think @brandonrosage should have a pick at this from the UI/X angle. I think it can be fairly easily done, but will require a clear distinction between OSM as the base-layer of physical objects and the Ushahidi layer of posts. That being said, it's also an improvement for users who may need to add a building, road, or re-name something but who don't want to do it themselves in OSM.

**Comment by aoduor** [comment]
@jshorland could I get a sense of where we're at with this - I'd spoken to the mapbox team about this and it would be great to share an update on what we're doing about it so far?

**Comment by jshorland** [comment]
@aodour I'm going to chat with @brandonrosage about this today. I don't *think* there's any sneaky design issues we should be concerned about, but want to double check with him before we send to devs. It should be quick to actually develop. 

The only other thing I'm concerned about is being able to track how many people actually use the link. I have a call scheduled with @rjmackay to speak with him about that as well. 

So basically, end of day we should know if we need MORE scoping/design, or if its ready to go to dev.

**Comment by aoduor** [comment]
Ping @jshorland :). Any update on this?

**Comment by jshorland** [comment]
@aoduor I'd suggest that instead of putting a link on the map itself, we add it to the location search tool if no results can be found. Thoughts?

**Comment by jshorland** [comment]
Scope:
- Add link to improve the baselayer on the map itself. Should be exactly like the example provided in Shadrock's link
